src/Form/ParticipationType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Form;
+
+use App\Entity\Conference;
+use App\Entity\Participation;
+use App\Repository\ConferenceRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ParticipationType extends AbstractType
+{
+    public function __construct(
+        private ConferenceRepository $conferenceRepository,
+    ) {
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('conference', EntityType::class, [
+                'class' => Conference::class,
+                'choice_label' => 'name',
+                'query_builder' => $this->conferenceRepository->getFutureConferences(),
```

Comment:
Please suffix the method `getFutureConferences` with `QueryBuilder`, to avoid confusion about returning data or actual QueryBuilder